config/locales/en.yml

```diff
@@ -273,6 +273,7 @@ en:
             email:
               blank: "Email can't be blank"
               taken: "Email is already registered"
+              invalid: "Invalid email"
```

Comment:
Given the way the message is created from this file we can just have the message as 'invalid' so it shows as 'Email invalid'

Reply:
Done